src/hooks/use-cart.ts

```diff
@@ -0,0 +1,28 @@
+import { useCartTotals, useCartData, useUpdateCartItemQuantity, useRemoveItemFromCart } from '~/api/api-hooks';
+import { useEcomAPI } from '~/api/ecom-api-context-provider';
+
+export const useCart = () => {
+    const ecomAPI = useEcomAPI();
+    const { data: cartData } = useCartData();
```

Comment:
maybe it would be better to rename values returned from hooks to not rename it during assignment?

Reply:
can't do that, because it's a `SWR` object returned from the hook